core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -758,7 +759,7 @@ func (ec *EthConfirmer) logFieldsPreviousAttempt(attempt EthTxAttempt) []interfa
 }
 
 func (ec *EthConfirmer) bumpGas(ctx context.Context, etx EthTx, previousAttempts []EthTxAttempt) (bumpedAttempt EthTxAttempt, err error) {
-	priorAttempts := make([]gas.PriorAttempt, len(previousAttempts))
+	priorAttempts := make([]gas.PriorAttempt[gas.EvmFee, common.Hash], len(previousAttempts))
```

Comment:
This priorAttempt here is inside generic component, and should not depend on the EvmFee.
Could we expose a function from the Estimator interface, which will create a PriorAttempt, and give it here in generic way?

Reply:
I'll leave a comment here

right now everything is still EVM specific, but in later PRs they will become generically typed once the top-level `TxManager` struct becomes generic